src/views/Results.vue

```diff
@@ -36,9 +36,12 @@
         <!-- Evaluation Info -->
         <v-row>
           <v-col xs-12>
-            <v-card elevation="2">
+            <v-card v-if="is_evaluation" elevation="2">
               <EvaluationInfo :filter="file_filter" />
             </v-card>
+            <v-card v-else elevation="2">
+              <ProfileInfo :filter="file_filter" />
+            </v-card>
```

Comment:
With the other changes here, this can be changed to:

```javascript
          <v-col xs-12 v-if="current_file">
            <v-card v-if="current_file.hasOwnProperty('evaluation')" elevation="2">
              <EvaluationInfo :filter="file_filter" />
            </v-card>
            <v-card v-if="current_file.hasOwnProperty('profile')" elevation="2">
              <ProfileInfo :profile="current_file" />
            </v-card>
          </v-col>
```